### DISCUSSION TITLE
how to add a trained module

### DISCUSSION THREAD

**Discussion by aeliavision** [erew123/alltalk_tts#342] [Q&A] [answered]
is it possible to use modules from online source 
like this one https://huggingface.co/sail-rvc/Britney_Spears__RVC_V2__500_Epochs/tree/main
how can i add it to alltalk ?

**Comment by MarianaMendanha** [comment]
yes, in ``alltalk_tts/models/rvc_voices directory`` add a folder with the name of preference of your model/voice, inside of your folder add the .pth and the .index that you can download from the url  (in your case: https://huggingface.co/sail-rvc/Britney_Spears__RVC_V2__500_Epochs/tree/main). If you didn't yet, enabled the RVC in you allTalk, in ``Global Settings`` -> ``Enable RVC``, refresh the server and your voice should be in the RVC options, like so:
![image](https://github.com/user-attachments/assets/fc44c815-4bc8-4f0f-a6c0-cc320991b144)

If something is not clear, you can always check the ``Documentation and Help`` tab.

**Reply by aeliavision** [reply]
but i cant see any of this settings , please see the screenshots
![Screenshot_13-9-2024_1617_127 0 0 1](https://github.com/user-attachments/assets/1ac7b31d-a3f6-4b4d-a769-726c6cb9c7e0)
![Screenshot_13-9-2024_16053_127 0 0 1](https://github.com/user-attachments/assets/8f5d8f34-6924-495e-8a8f-1a22f9c45ae4)

**Comment by MarianaMendanha** [comment, marked as answer]
You are using version 1 of AllTalk. I am not familiar with it. I advise you to download the v2 BETA here: https://github.com/erew123/alltalk_tts/tree/alltalkbeta
